README.md

```diff
@@ -66,6 +81,10 @@ AWS_SECRET_ACCESS_KEY=<provided-on-request>
 AWS_BUCKET_NAME=<provided-on-request>
 AWS_DEFAULT_REGION=<provided-on-request>
 
+# CDS API credentials
```

Comment:
Can you make sure these are added to the Job Compute and Personal Compute policies? Also to the shared `init-dev-base.sh`.

Reply:
Looks like you beat me to it and already added those!